code/modules/power/singularity/emitter.dm

```diff
@@ -567,3 +612,58 @@
 	req_access = list("science")
 	welded = TRUE
 	use_power = NO_POWER_USE
+
+/obj/item/emitter_disk
+	name = "Diode Disk: Debugger"
+	desc = "This disk can be used on an emitter with an open panel to reset its projectile. Unless this was handed to you by an admin, you should report this on github."
+	icon = 'icons/obj/devices/circuitry_n_data.dmi'
+	icon_state = "datadisk6"
+	var/stored_proj = /obj/projectile/beam/emitter/hitscan
+	var/stored_sound = 'sound/items/weapons/emitter.ogg'
+	var/consumed_on_removal = TRUE
+	var/consumable = TRUE
+	var/fire_rate_mod = 1
+	var/no_shot_counter = FALSE
+
+/obj/item/emitter_disk/stamina
+	name = "Diode Disk: Electrodisruptive"
```

Comment:
```suggestion
	name = "/improper Diode Disk: Electrodisruptive"
```
Put this in front of any object names that start with capitalization (but not datum), as this is an improper noun. If this isn't here, it will treat it as a proper noun and will cause chat messages to be grammatically incorrect.